torch/onnx/verification.py

```diff
@@ -203,6 +220,17 @@ def _onnx_reference_evaluator_session(model: Union[str, io.BytesIO]):
     return onnx_session
 
 
+@_beartype.beartype
+def _onnx_backend_session(model: Union[str, io.BytesIO], backend: OnnxBackend):
```

Comment:
nit: Return type?

Reply:
So far on module level we don't have dependency on onnx or onnxruntime. Any idea how to annotate return type?